en/development/configuration.rst

```diff
@@ -51,7 +51,7 @@ datasource
 persistent
     Whether or not to use a persistent connection to the database. When using
     SQLServer you should not enable persistent connections as it causes
-    difficult to diagnose crashes.
+    difficulty to diagnose crashes.
```

Comment:
Could also do _you should not enable persistent connections, because they create difficult to diagnose crashes_ as well.